gradle.properties

```diff
@@ -49,7 +49,7 @@ neoforge_data_pack_format=18
 mod_version=1.0.1
 license=https://github.com/CommandrMoose/TardisRefined/blob/master/LICENSE
 credits=Commoble, Lucraft, starray1000000
-mod_authors=CommandrMoose, Suff99, Magic Man, ILikePandas, Jacob K, 50ap5ud5
+mod_authors=CommandrMoose, Jeryn99, Magic Man, ILikePandas, Jacob K, 50ap5ud5
 mod_description=TARDIS: Refined is a Minecraft mod based on the world of Doctor Who that adds a personal, teleporting base to the game. The mod explores the nature of the TARDIS and how it operates and expresses its personality.
 display_url=https://github.com/CommandrMoose/TardisRefined
```

Comment:
Might want to update it here too whilst you're at it.

Reply:
Consider it done, same goes for Modrinth and Curse